### PR TITLE
Adding mocked email config

### PR DESCRIPTION
**Pull request by Sithi5**

### CODE REVIEW

File: parsec/backend/cli/run.py [hunk 1]
```diff
@@ -471,8 +497,11 @@ async def _serve_client(stream):
                 await trio.serve_tcp(_serve_client, port, host=host)
 
         click.echo(
-            f"Starting Parsec Backend on {host}:{port} (db={config.db_type}, "
-            f"blockstore={config.blockstore_config.type})"
+            f"Starting Parsec Backend on {host}:{port}"
+            f" (db={config.db_type}"
+            f" blockstore={config.blockstore_config.type}"
+            f" backend_addr={config.backend_addr}"
+            f" email_config={email_config.__class__.__name__})"
```
Comment (vxgmichel) [inline parsec/backend/cli/run.py:504]: There might be a better way of representing the email config? Maybe use `str(email_config)` and define the `__str__` method for `SmtpEmailConfig` et `MockedEmailConfig`?

File: parsec/backend/invite.py [hunk 2]
```diff
@@ -196,6 +198,27 @@ def _do():
...
+async def _mocked_send_mail(
+    email_config: MockedEmailConfig, to_addr: str, message: Message
+) -> None:
+    def _do():
+        tmpfile_fd, tmpfile_path = tempfile.mkstemp(
+            prefix="tmp-email-", suffix=".html", dir=email_config.tmpdir
+        )
+        tmpfile = open(tmpfile_path, "w")
+        tmpfile.write(message.as_string())
+        print("You can access to the mail file here : ", tmpfile.name, file=sys.stderr)
```
Comment (vxgmichel) [inline parsec/backend/invite.py:210]: Maybe something like: 
```
f"""\
A request to send an e-mail to {to_addr} has been triggered and mocked. 
The mail file can be found here: {tmpfile.name}"""`

Comment (Sithi5) [reply parsec/backend/invite.py:210]: Done

File: parsec/backend/cli/run.py (hunk 1 above)
Comment (Sithi5) [reply parsec/backend/cli/run.py]: Done :)

**Pull request merged**